stdlib/sqlite3/dbapi2.pyi

```diff
@@ -253,3 +297,15 @@ if sys.version_info < (3, 8):
         def __init__(self, *args, **kwargs): ...
 
 class Warning(Exception): ...
+
+if sys.version_info >= (3, 11):
+    class Blob:
+        def close(self) -> None: ...
+        def read(self, __length: int = ...) -> bytes: ...
+        def write(self, __data: bytes) -> None: ...
+        def tell(self) -> int: ...
+        # whence must be one of os.SEEK_SET, os.SEEK_CUR, os.SEEK_END
+        def seek(self, __offset: int, __whence: int = ...) -> None: ...
+        def __len__(self) -> int: ...
+        def __enter__(self: Self) -> Self: ...
+        def __exit__(self, __typ: object, __val: object, __tb: object) -> Literal[False]: ...
```

Comment:
I think this is going to anger my unreleased Y036 check over at flake8-pyi :/ https://github.com/PyCQA/flake8-pyi/commit/b7f7e9a7cfab24862ae8bf0f620ad8b84e740d96

Reply:
hm, do you think we should change it? This `__exit__` method really does take `object`; that's what the AC annotations say.

Reply:
'Kay, I'll work up a patch for flake8-pyi.